Packages/MIES/MIES_AmplifierInteraction.ipf

```diff
@@ -1455,6 +1455,11 @@ Function AI_FillAndSendAmpliferSettings(panelTitle, sweepNo)
 		clampMode = DAG_GetHeadstageMode(panelTitle, i)
 		AI_AssertOnInvalidClampMode(clampMode)
 
+		STRUCT AxonTelegraph_DataStruct tds
+		AI_InitAxonTelegraphStruct(tds)
+		AxonTelegraphGetDataStruct(axonSerial, channel, 1, tds)
+		ASSERT(clampMode == tds.OperatingMode, "A clamp mode mismatch was detected. Please report that as a bug with a description what you did. Thanks!")
```

Comment:
@t-b does this get run if Syncronize MIES to MCC is unchecked?

Reply:
Yes this is always checked, "Syncronize MIES to MCC" does not play a role in that check here.

Reply:
I'm not saying this is the perfect fix for the issue. I've asked luke already for the pxp so that I can have a look at the history. History saving did not work somehow, I've opened an issue for that.